image/Dockerfile

```diff
@@ -18,17 +25,45 @@ RUN apt-get update && \
     g++ \
     libsm6 \
     zlib1g-dev \
-    libopenexr-dev && \
-  /golem/install_py_libs.sh /golem/work/requirements.txt && \
-  /golem/copy.sh && \
-  apt-get remove -y \
-    libopenexr-dev \
-    zlib1g-dev \
-    wget \
-    libxrender1 \
-    libsm6 \
-    g++ \
-    libglib2.0-0 && \
+    libopenexr-dev
+
+COPY --from=blender /blender /blender
+# Check to see that all libraries are available
+RUN ! ldd /blender/blender | grep "=> not found"
+
+COPY entrypoints/scripts/verifier_tools/requirements.txt /tmp
+RUN apt-get install -y python3-pip
+RUN pip3 install --upgrade pip
+RUN pip install -r /tmp/requirements.txt
```

Comment:
is this using the right version of pip? it looks like python2

Reply:
Yeah, this is taken from `install_py_libs.sh` and apparently after you run `pip3 install --upgrade pip` `pip` starts to link to `pip3`.